### DISCUSSION TITLE
resolve.aliasFields option

### DISCUSSION THREAD

**Discussion by jianxingyao** [webpack/webpack#18960] [General]
resolve.aliasFields option No matter how I set it up, nothing has changed for my project. I thought mainFields and exportsFields were overridden with too high priority, I tried to set these two fields to empty arrays, aliasFields didn't take effect either, what the hell does it do!?

**Comment by alexander-akait** [comment]
The `exports` field have higher priority

**Reply by jianxingyao** [reply]
Yes, so I set both mainFields and exportsFields as empty arrays, and package.json only have the main and browser fields to set the entrances, but still aliasFields doesn't work

**Reply by alexander-akait** [reply]
why do you need this field?

**Reply by jianxingyao** [reply]
I would like to know when this field will take effect, and what is special about it compared to the other two, that's all. If both mainFields and exportsFields have a higher priority than aliasFields, does that mean that this field can be discarded?

**Reply by alexander-akait** [reply]
There is an example of what it is needed for -  https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L1547

**Reply by jianxingyao** [reply]
Thank you!
